repos.d/gentoo/overlays.yaml

```diff
@@ -1,3 +1,34 @@
+###########################################################################
+# GURU
+###########################################################################
+- name: gentoo_ovl_guru
+  type: repository
+  desc: Gentoo overlay GURU
+  family: gentoo
+  color: '62548f'
+  minpackages: 100
+  default_maintainer: guru@gentoo.org
+  sources:
+    - name: guru-overlay
+      fetcher: GitFetcher
+      parser: GentooGitParser
+      require_md5cache_metadata: false
```

Comment:
Please enable this back. Repology requires information on homepages and downloads which are stored in metadata/md5cache.

Reply:
Done. Change to `true` for `require_md5cache_metadata`

Reply:
@AMDmi3 ,
is `require_xml_metadata` could be left `false` in this case? 
The information about gentoo packages is stored in `metadata.xml` files